signing_service/signing_service/signing_service.py

```diff
@@ -578,4 +596,9 @@ def _parse_arguments() -> argparse.Namespace:
         args.signing_service_private_key,
         args.ethereum_private_key,
         args.max_reconnect_attempts,
+        Notifier(
```

Comment:
Won't it be better to start `Notifier` after connection to `MiddleMan` is established? @dybi what do you think?

Reply:
The question is, are we planning to use email Notifications somewhere else in near future other then `_handle_connection`.

Reply:
@rwrzesien, I wouldn't be for tangling `Notifier` and `SigningService`classes. Right know we have loose coupling between them and `Notifier` isn't even aware of existence of `SignigService` and `SignigService` just gets something in its `__init__`  and knows that it use `send(message)` of thath something ;) Let it stay as it is.